syntax = "proto2";

import "maid/controller.proto";

package maid.proto;

message ConnectionProto
{
	optional int64 id = 1;
	optional string host = 2;
	optional int32 port = 3;

	extensions 1000 to max;
}

extend ControllerProto
{
	optional ConnectionProto connection = 1001;
}
